axelor-account/src/main/java/com/axelor/apps/account/service/invoice/InvoiceServiceImpl.java

```diff
@@ -123,6 +132,8 @@ public InvoiceServiceImpl(
     this.partnerService = partnerService;
     this.invoiceLineService = invoiceLineService;
     this.accountConfigService = accountConfigService;
+    this.partnerTurnoverServiceImpl = partnerTurnoverServiceImpl;
+    this.yearServiceAccountImpl = yearServiceAccountImpl;
```

Comment:
Please avoid injection implementations. You can either add the method you need in YearService interface or create a new interface YearAccountService with the new method (and bind YearAccountService to YearServiceAccountImpl)

Reply:
Done